app/helpers/home_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```

Comment:
I think we can remove this as well for now.

Reply:
Done